ratpack/src/main/groovy/com/baeldung/Ratpack.groovy

```diff
@@ -0,0 +1,61 @@
+package com.baeldung;
+
+@Grab('io.ratpack:ratpack-groovy:1.6.1')
+import static ratpack.groovy.Groovy.ratpack
+
+import com.baeldung.model.User
+import com.google.common.reflect.TypeToken
+import ratpack.exec.Promise
+import ratpack.handling.Context
+import ratpack.jackson.Jackson
+import groovy.sql.Sql
+import ratpack.hikari.HikariModule
+
+ratpack {
+    bindings {
+        module(HikariModule) { config ->
+            config.dataSourceClassName = 'org.h2.jdbcx.JdbcDataSource'
+            config.addDataSourceProperty('URL', "jdbc:h2:mem:devDB;INIT=RUNSCRIPT FROM 'classpath:/User.sql'")
+        }
+    }
+
+
+    handlers {
+        
+        get {
+            render 'Hello World from Ratpck with Groovy!!'
+        }
+        
+        get("greet/:name"){Context ctx ->
```

Comment:
Watch the whitespace here between `{}` and `()`

Reply:
will correct it